nimare/meta/cbma/ale.py

```diff
@@ -151,6 +152,18 @@ def __init__(
 
     def _compute_summarystat_est(self, ma_values):
         stat_values = 1.0 - np.prod(1.0 - ma_values, axis=0)
+
+        # np.array type is used by _determine_histogram_bins to calculate max_poss_ale
+        if isinstance(stat_values, sparse._coo.core.COO):
+            masker = self.dataset.masker if not self.masker else self.masker
+            mask_data = masker.mask_img.get_fdata().astype(bool)
+
+            stat_values = stat_values.todense().reshape(-1)  # Indexing a .reshape(-1) is faster
+            stat_values = stat_values[mask_data.reshape(-1)]
+
+            # This is used by _compute_null_approximate
+            self.n_mask_voxels = stat_values.shape[0]
```

Comment:
```suggestion
            self.__n_mask_voxels = stat_values.shape[0]
```
I'm thinking this should either be private (leading underscores) or considered a fitted, data-dependent parameter (trailing underscore).